lib/pbench/cli/server/shell.py

```diff
@@ -72,19 +72,13 @@ def generate_crontab_if_necessary(
     return ret_val
 
 
-def main():
+def run_gunicorn(server_config: PbenchServerConfig, logger: Logger) -> int:
     """Setup of the Gunicorn Pbench Server Flask application.
 
-    If an error is encountered during setup, system exit is invoked with an
-    exit code of 1.  Otherwise, the exit code of the gunicorn subprocess is
-    used.
+    Returns:
+        0 on success, 1 on error, and passes along the gunicorn sub-process
```

Comment:
The "and" implies returning two values. Really I think we're returning non-zero (e.g., the non-zero return from the crontab generation) if we hit a "local" error before running `gunicorn` and otherwise returning the `gunicorn` status: which may be 1, 0, or anything else it pleases.

Reply:
Fixed.